index.html

```diff
@@ -302,7 +302,30 @@ <h2 style="font-family: cursive;">
             name="pns"
             onclick="window.pointAndSwap=!window.pointAndSwap; window.tempS = undefined;"
           />
-          <label for="pns">Point & Swap mode</label>
+          <label for="pns">Point &amp; Swap mode</label>
+          <br />
+
+          <input
+            type="button"
+            value="-"
+            onclick="updateExtraDims(document.getElementById('xsz'), -1)"
+          />
+          <input
+            autocomplete="off"
+            type="textbox"
+            value="0"
+            id="xsz"
+            name="xsz"
+            size="1"
+            onblur="updateExtraDims(this, 0)"
+          />
+          <input
+            type="button"
+            value="+"
+            onclick="updateExtraDims(document.getElementById('xsz'), 1)"
+          />
+          <label>Extra dimensions</label>
```

Comment:
nit: please consider some renaming similar to the following:
"Extra dimensions" -> "Size Offset"
"updateExtraDims" -> "updateSizeOffset"
"extraDims" -> "sizeOffset"

The reason for this is that the term "extra dimension" generally is used for objects such as the tesseract, which require extra dimensions orthogonal to the triple (x,y,z) to exist.
reference: https://superliminal.com/cube/cube.htm

Reply:
Gladly. Thanks for suggesting the alternate names.